src/guides/v2.4/migration/migration-tool-internal-spec.md

```diff
@@ -165,6 +165,42 @@ Change prefix option in case Magento was installed with prefix in database table
 
 Configuration data is accessible via \Migration\Config class.
 
```

Comment:
Repeat all corrections to the 2.3 version on the 2.4 version. Will double check 2.4